modules/caffeCoder/src/CaffeFeatExtractor.hpp

```diff
@@ -19,17 +19,17 @@
 #include "caffe-version.h"
 
 #if (CAFFE_MAJOR >= 1)
-	// new caffe headers
-	#include "caffe/caffe.hpp"
- 	#include "caffe/layers/memory_data_layer.hpp"
+    // new caffe headers
+    #include "caffe/caffe.hpp"
+    #include "caffe/layers/memory_data_layer.hpp"
 #else
```

Comment:
I think we can get rid of this, as we decided to require Caffe version 1 or bigger.